simdepot.cc

```diff
@@ -351,7 +351,9 @@ convoihandle_t depot_t::copy_convoi(convoihandle_t old_cnv, bool local_execution
 		new_cnv->set_name(old_cnv->get_internal_name());
 		int vehicle_count = old_cnv->get_vehicle_count();
 		for (int i = 0; i<vehicle_count; i++) {
-			const vehicle_desc_t * info = old_cnv->get_vehikel(i)->get_desc();
+			int i_cor = i;
+			if (old_cnv->is_reversed()){i_cor=vehicle_count-i-1;}
```

Comment:
`cor` はコードベース中でcommonな略語ではないので使用しないでください。（第三者が意味を理解できません）
三項演算子でまとめちゃった方が逆に読みやすいと思います。
```suggestion
			const int vehicle_index = old_cnv->is_reversed() ? vehicle_count - i - 1 : i; // Always copy in the normal order
```